mutiny-wasm/src/indexed_db.rs

```diff
@@ -270,6 +268,20 @@ impl IndexedDbStorage {
                     None => return Ok(Some((kv.key, obj.value))),
                 }
             }
+            DEVICE_LOCK_KEY => {
```

Comment:
Why get the device key from the remote server? We shouldn't be replacing our remote key, right? We just want to compare our device id from the device id from remote?

Reply:
This isn't the device id but the struct containing the currently locked device + time it was locked.